src/TuDa.CIMS.Web/Components/FilterComponent.razor

```diff
@@ -0,0 +1,77 @@
+﻿@* FilterComponent *@
+
+<!-- Frame -->
+<form>
+    <div class="p-3 bg-light rounded" style="width: 250px;">
+
+
+        <!-- Keyword Tags -->
+        <h6>Keywords</h6>
+        <div class="d-flex flex-wrap gap-2 mb-3">
+            @foreach (var keyword in Keywords)
+            {
+                <span class="badge bg-secondary">
+                    @keyword
+                    <button class="btn-close btn-close-white btn-sm ms-2"
+                            @onclick="() => RemoveKeyword(keyword)">
```

Comment:
🔧 Sorry, I made a mistake with the formatting. We can format it like this, but the way it's mentioned in #19 is easier to read IMO
```suggestion
                    <button 
                        class="btn-close btn-close-white btn-sm ms-2"
                        @onclick="() => RemoveKeyword(keyword)"
                    >
```